modules/analytics/src/main/java/com/intuit/wasabi/analytics/impl/ExperimentDetailsImpl.java

```diff
@@ -89,7 +89,7 @@ public ExperimentDetailsImpl(@CassandraRepository ExperimentRepository cassandra
      * @return the same ExperimentDetail object but with additional information
      */
     private ExperimentDetail getBucketData(ExperimentDetail exp) {
-        List<Bucket> buckList = buckets.getBuckets(exp.getId()).getBuckets();
+        List<Bucket> buckList = buckets.getBuckets(exp.getId(), false).getBuckets();
```

Comment:
I think changing the name of the method to getBucketsWith/WithoutExp check is more comprehensible than true/false argument.